pkg/apis/kab/v1alpha1/types.go

```diff
@@ -103,8 +103,12 @@ func NewManifest(path string) (manifest *Manifest, err error) {
 }
 
 // Embeds the contents of Path url into Content field for all resources
+// only when there is no previous content.
 func (m *Manifest) InlineContent() error {
 	err := m.PatchResourceContent(func(res *KabResource) (string, error) {
+		if res.Content != "" {
+			return res.Content, nil
+		}
```

Comment:
perhaps also blank out `.Path` to prevent accidental usage

Reply:
To help us resolve where the `.Content` came from, I appended "-invalid" to `.Path` so that it won't resolve, but we will still have it around.

Reply:
or flip it around to a prefix so you can be certain is will never resolve.

`kab-resolved:<previous-path>`

Reply:
fixed.